e2e-tests/playwright/tests/functional/system_console/sytem_users/export_data.spec.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {expect} from '@playwright/test';
+
+import {test} from '@e2e-support/test_fixture';
+import { duration } from '@e2e-support/util';
+
+test('MM-X should begin export of data when export button is pressed', async ({pw, pages}) => {
```

Comment:
@M-ZubairAhmed This test will timeout, as we are waiting for 2 reports to finish. There isn't really a meaningful way to break these up so that it runs quicker, so do you know if we can increase the timeout per test?

Reply:
As commented in the below, if we split out to another test. In this test we can just wait for the DM to have that particular post.
We currently have custom wait function in `utils` with a big warning not to use :P But i believe we probably wont be needing that because we are utilizing `waitUntilLastPostContains` can we can pass custom timeout here instead

Reply:
@saturninoabril Is there a way we can increase the timeout per test from 60 seconds?

Reply:
Not sure if this is what you're looking for but there is `test.setTimeout(duration.one_min)` or `test.slow()`. See https://playwright.dev/docs/test-timeouts#set-timeout-for-a-single-test.